src/formik.tsx

```diff
@@ -639,7 +619,7 @@ function warnAboutMissingIdentifier({
  * Transform Yup ValidationError to a more usable object
  */
 export function yupToFormErrors<Values>(yupError: any): FormikErrors<Values> {
```

Comment:
yup is now definitively typed :) ... take a look https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/yup/index.d.ts

Reply:
Yah will do